django_etebase/exceptions.py

```diff
@@ -12,37 +10,42 @@ def __init__(self, code, detail, status_code=status.HTTP_400_BAD_REQUEST):
         self.status_code = status_code
 
 
-class AuthenticationFailed(HTTPException):
+class CustomHttpException(Exception):
+    def __init__(self, detail: str, status_code: int, code: str):
+        self.status_code = status_code
+        self.code = code
+        self.detail = detail
+
+    @property
+    def as_dict(self) -> dict:
+        return {"code": self.code, "detail": self.detail}
+
+
+class AuthenticationFailed(CustomHttpException):
     def __init__(
         self,
+        detail: str = "Incorrect authentication credentials.",
         status_code: int = status.HTTP_401_UNAUTHORIZED,
```

Comment:
btw, `status` is defined in DRF which we will remove. Maybe better to define our own constants or our own `status`?

Reply:
its already imported from fastapi 
```python
from fastapi import status
```

Reply:
Sorry for the noise.